docs/source/scripts/exportdb/blurb.md

```diff
@@ -0,0 +1,9 @@
+ExportDB is a centralized dataset of supported and unsupported export cases.
+This is targeted towards users who want to understand specifically what types of
+code are supported, the subtleties of export, and how to modify their existing
+code to be compatible with export.  Note that this is not an exhaustive set of
+everything that is supported and unsupported by export, but will hopefully cover
+most common and confusing use cases that users will run into.
+
+If you have a feature that you think needs a stronger guarantee from us to
+support in export please reach out to us!
```

Comment:
how users can reach out? Create an issue with the correct tag in the pytorch/pytorch repo?

Reply:
Thanks for the feedback! I updated it with asking users to tag module:export.